Editor/Resources/EditorWindow/Pages/HelpAndDocumentationPage.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using AmazonGameLift.Editor;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+namespace Editor.Resources.EditorWindow.Pages
+{
+    public class HelpAndDocumentationPage
+    {
+        private readonly VisualElement _container;
+
+        public HelpAndDocumentationPage(VisualElement container)
+        {
+            _container = container;
+            var mVisualTreeAsset = UnityEngine.Resources.Load<VisualTreeAsset>("EditorWindow/Pages/HelpAndDocumentationPage");
+            var uxml = mVisualTreeAsset.Instantiate();
+
+            container.Add(uxml);
+            ApplyText();
+            
+            // _container.Q<Button>("CreateAccount").RegisterCallback<ClickEvent>(_ => onCreateAccountClicked());
+            // _container.Q<Button>("AddProfile").RegisterCallback<ClickEvent>(_ => onAddProfileClicked());
+            // _container.Q<Button>("DownloadSampleGame").RegisterCallback<ClickEvent>(_ => onImportSampleClicked());
```

Comment:
Are these to be added in later? Please remove for now, and re-add as and when required.  Alternatively, I would drop in a comment of the nature to ensure this is picked up later. 

```
// TODO: ASG-8
```